apiary_v2.apib

```diff
@@ -881,7 +884,11 @@ follows:
 ## Create an Order [POST]
 
 Once an order has been placed on the ecommerce platform, the Unmade system should be notified to begin the process of 
-generating manufacturing files.
+generating manufacturing files. To do this, pass a unique order reference, a shipping address and one or more order
+items to this endpoint via an HTTP POST request. Each item must reference an Unmade Design ID.
+
+Optionally, arbitrary key-value pairs can be passed as a `partner_data` object at both the order and order item level. 
+This may be used by your system to provide your internal ID or similar reference for this order.
```

Comment:
Could we say something along the lines of "... pass information to your vendor ..." (this is how they use it mostly)